### PR TITLE
Internal: Better handling safe-mode cookie [ED-7105]

### PR DESCRIPTION
**Pull request by nahmana**

### CODE REVIEW

File: modules/safe-mode/module.php [hunk 1]
```diff
@@ -101,13 +101,13 @@ public function enable_safe_mode() {
...
 
-		$token = md5( wp_rand() );
+		$token = hash( 'sha256', wp_rand() );
 
 		// Only who own this key can use 'elementor-safe-mode'.
 		update_option( self::OPTION_TOKEN, $token );
 
 		// Save for later use.
-		setcookie( self::OPTION_TOKEN, $token, time() + HOUR_IN_SECONDS, COOKIEPATH );
+		setcookie( self::OPTION_TOKEN, $token, time() + HOUR_IN_SECONDS, COOKIEPATH, '', true, true );
```
Comment (StyleShit) [inline modules/safe-mode/module.php:110]:
```suggestion
		setcookie( self::OPTION_TOKEN, $token, time() + HOUR_IN_SECONDS, COOKIEPATH, '', is_ssl(), true );
```

File: modules/safe-mode/module.php [hunk 2]
```diff
@@ -131,7 +131,7 @@ public function disable_safe_mode() {
 		delete_option( 'elementor_safe_mode_created_mu_dir' );
 
 		delete_option( self::OPTION_TOKEN );
-		setcookie( self::OPTION_TOKEN, '', 1 );
+		setcookie( self::OPTION_TOKEN, '', 1, '', '', true, true );
```
Comment (StyleShit) [inline modules/safe-mode/module.php:134]: same as above

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: modules/safe-mode/module.php (hunk 1 above)
Comment (nahmana) [reply modules/safe-mode/module.php:110]: Done.

File: modules/safe-mode/module.php (hunk 2 above)
Comment (nahmana) [reply modules/safe-mode/module.php:134]: done.

**Review by StyleShit — APPROVED** [review]

**Pull request merged**